apps/pie-storybook/stories/pie-textarea.stories.ts

```diff
@@ -6,7 +6,7 @@ import { useArgs as UseArgs } from '@storybook/preview-api';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-textarea';
 import {
-    TextareaProps, defaultProps, resizeModes, sizes,
+    TextareaProps, defaultProps, resizeModes, sizes, statusTypes,
```

Comment:
can `type` be added to any of these imports if they are not values? It improves the compiled JS

Reply:
same Q for any other imports in this PR

Reply:
`resizeModes`, `sizes`, `statusTypes` - they are values used for storybook controls options